.github/workflows/ai-service-release-nightly-image.yaml

```
@@ -6,43 +6,95 @@ on:
-          push: true
-          platforms: linux/amd64,linux/arm64
-          tags: ${{ steps.meta.outputs.tags }}
-          context: ./wren-ai-service
-          file: ./wren-ai-service/docker/Dockerfile
\ No newline at end of file
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Create manifest list and push
+        working-directory: /tmp/digests
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:2:33: Quote this to prevent word splitting

(shellcheck)